common/extensions/api/brave_rewards.json

```diff
@@ -396,14 +396,18 @@
             "type": "integer"
           },
           {
-            "name": "tweetMetaData",
+            "name": "mediaMetaData",
             "type": "object",
             "properties": {
+              "mediaType": {
+                "type": "string",
+                "description": "description used in UI to identify the media type"
+              },
               "userId": {
                 "type": "string",
                 "description": "User ID of tweet author"
               },
-              "name": {
+              "twitterName": {
                 "type": "string",
                 "description": "Name of tweet author"
```

Comment:
Does this description need an update as well?

Reply:
I don't think so. This is still going to a twitter designated function.

Reply:
I see, I misread that diff!